app/models/post.rb

```diff
@@ -1,11 +1,15 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :user, foreign_key: 'user_id'
```

Comment:
- You should remove this also to keep your previous code which is consistent following the ERD structure for the database.